pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/cronjob_v1_test.go

```diff
@@ -24,6 +24,7 @@ import (
 func TestExtractCronJobV1(t *testing.T) {
 	creationTime := metav1.NewTime(time.Date(2021, time.April, 16, 14, 30, 0, 0, time.UTC))
 	lastScheduleTime := metav1.NewTime(time.Date(2021, time.April, 16, 14, 30, 0, 0, time.UTC))
+	lastSuccessfulTime := metav1.NewTime(time.Date(2021, time.April, 16, 14, 30, 0, 0, time.UTC))
```

Comment:
nit: I know this file is already using the same dates above, but I prefer using unique dates for each attribute as this code is primary testing model translation, and picking the wrong date field is likely, as humans

Reply:
Thanks, fixed it https://github.com/DataDog/datadog-agent/compare/4e6c8a23bd5405aea0aa2f22ef45435f5a79701e..1fbe09b8884234edb5d5738fcf169413e74db39f